src/_locales/es-ES.json

```diff
@@ -131,22 +131,22 @@
   "settings-options-style-dialog-button": "Cargar CSS",
   "settings-options-style-dialog-css-files": "Ficheros CSS",
   "settings-options-style-dialog-all-files": "Todos los Ficheros",
-  "slack-token-label": "",
-  "slack-token-placeholder": "",
+  "slack-token-label": "Ingresa tu token de usuario slack (puedes encontrarlo en https://api.slack.com/custom-integrations/legacy-tokens)",
+  "slack-token-placeholder": "Token Slack",
 
   "title-color-picker": "Selector de Color",
   "title-settings": "Ajustes",
   "title-settings-playback": "Reproducción",
   "title-settings-general": "General",
-  "title-settings-hotkeys": "Atajos de Teclado",
+  "title-settings-hotkeys": "Atajos",
   "title-settings-lastfm": "LastFM",
   "title-settings-listenbrainz": "ListenBrainz",
-  "title-settings-mini": "Reproductor Mini",
-  "title-settings-style": "Estilos Personalizados",
-  "title-settings-slack": "",
+  "title-settings-mini": "Mini",
```

Comment:
Would "Mini Player" work in Spanish? In Portuguese it's common to use the English word if the native translation is awkward.

Reply:
Yes, in spanish also. So, I think that we can change this without problem.